src/TLuaInterpreter.cpp

```diff
@@ -15027,52 +15028,66 @@ void TLuaInterpreter::loadGlobal()
 }
 
 // No documentation available in wiki - internal function
-int TLuaInterpreter::startPermTimer(const QString& name, const QString& parent, double timeout, const QString& function)
+QPair<int, QString> TLuaInterpreter::startPermTimer(const QString& name, const QString& parent, double timeout, const QString& function)
 {
-    QTime time(0, 0, 0, 0);
-    auto msec = static_cast<int>(timeout * 1000);
-    QTime time2 = time.addMSecs(msec);
+    QTime time = QTime(0,0,0,0).addMSecs(qRound(timeout * 1000));
     TTimer* pT;
     if (parent.isEmpty()) {
-        pT = new TTimer("a", time2, mpHost);
+        pT = new TTimer(QStringLiteral("newPermTimerWithoutAnId"), time, mpHost);
     } else {
-        TTimer* pP = mpHost->getTimerUnit()->findTimer(parent);
-        if (!pP) {
-            return -1; //parent not found
+        // NOTE: There can be more than one timer with the same name - we will
+        // use only the FIRST one:
+        QList<TTimer*> timerList = mpHost->getTimerUnit()->findTimer(parent);
+        if (timerList.isEmpty()) {
+            return qMakePair(-1, QStringLiteral("parent \"%1\" not found").arg(parent)); //parent not found
         }
-        pT = new TTimer(pP, mpHost);
+        pT = new TTimer(timerList[0], mpHost);
     }
-
-    pT->setTime(time2);
+    pT->setTime(time);
     pT->setIsFolder(false);
     pT->setTemporary(false);
-    pT->registerTimer();
-    pT->setScript(function);
+    // DE: darf erst nach isTempTimer gesetzt werde, damit setName() schneller ist
```

Comment:
We should delete the German comment, no need to keep it around

Reply:
Just the German - or both?

Reply:
Just the German one

Reply:
Done.